server/src/Api/Handlers.hs

```diff
@@ -46,6 +67,57 @@ hashScript :: HashScriptRequest -> AppM HashedScript
 hashScript (HashScriptRequest script) =
   pure . HashedScript $ hashLedgerScript script
 
+finalizeTx :: FinalizeRequest -> AppM FinalizedTransaction
+finalizeTx req@(FinalizeRequest {tx, datums, redeemers}) = do
+  pparams <- asks protocolParams
+  liftIO $ do
```

Comment:
Are these `print`s and `putStrLn`s just for debugging (I mean here and elsewhere)?

Reply:
Yes

Reply:
Should I remove them?

Reply:
I would say so, when you're done debugging